comixed-release/src/main/assembly/scripts/run.sh

```diff
@@ -112,7 +112,8 @@ fi
 # build a list of JVM arguments
 
 if [[ $LIBDIR ]]; then
-  JVMOPTIONS="${JVMOPTIONS} -classpath ${LIBDIR}"
+  JVMOPTIONS="$JVMOPTIONS -classpath"
+  CLASSPATH="${LIBDIR}"
 fi
 
-$JAVA ${JVMOPTIONS} -jar ${COMIXED_JAR_FILE} ${JAROPTIONS}
+$JAVA ${JVMOPTIONS} "${CLASSPATH}" -jar "${COMIXED_JAR_FILE}" ${JAROPTIONS}
```

Comment:
As noted above, if `LIBDIR` didn't exist, `JVMOPTIONS` is unchanged and `CLASSPATH` is an empty string.

If `LIBDIR` does exist, `JVMOPTIONS` suffixes the `-classpath` and here I can enclose `CLASSPATH` (`LIBDIR`) in quotes to properly handle directory paths with spaces in them.